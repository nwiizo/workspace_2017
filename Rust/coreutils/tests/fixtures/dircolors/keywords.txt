NORMAL 00 # no color code at all
FILE 00 # regular file: use no color at all
RESET 0 # reset to "normal" color
Dir 01;34 # directory
LINK 01;36 # symbolic link. (If you set this to 'target' instead of a
multihardlink 00 # regular file with more than one link
fifo 40;33 # pipe
SOCK 01;35 # socket
DOOR 01;35 # door
BLK 40;33;01 # block device driver
CHR 40;33;01 # character device driver
orphan 40;31;01 # symlink to nonexistent file, or non-stat'able file ...
MISSING 00 # ... and the files they point to
setuid 37;41 # file that is setuid (u+s)
SETGID 30;43 # file that is setgid (g+s)
CAPABILITY 30;41 # file with capability
STICKY_OTHER_WRITABLE 30;42 # dir that is sticky and other-writable (+t,o+w)
Other_writable 34;42 # dir that is other-writable (o+w) and not sticky
STICKY 37;44 # dir with the sticky bit set (+t) and not other-writable
EXEC 01;32
LEFTCODE    \e[
RIGHTCODE   m
ENDCODE \xff
