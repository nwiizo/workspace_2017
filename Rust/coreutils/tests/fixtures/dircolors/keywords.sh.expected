LS_COLORS='no=00:fi=00:rs=0:di=01;34:ln=01;36:mh=00:pi=40;33:so=01;35:do=01;35:bd=40;33;01:cd=40;33;01:or=40;31;01:mi=00:su=37;41:sg=30;43:ca=30;41:tw=30;42:ow=34;42:st=37;44:ex=01;32:lc=\e[:rc=m:ec=\xff:';
export LS_COLORS
